x-pack/solutions/security/plugins/elastic_assistant/server/ai_assistant_service/helpers.ts

```diff
@@ -16,6 +16,7 @@ import type { Logger } from '@kbn/logging';
 import { getResourceName } from '.';
 import { knowledgeBaseIngestPipeline } from '../ai_assistant_data_clients/knowledge_base/ingest_pipeline';
 import { GetElser } from '../types';
+import { ELASTICSEARCH_ELSER_INFERENCE_ID } from '../ai_assistant_data_clients/knowledge_base/field_maps_configuration';
```

Comment:
Can we remove this `const` and prefer to always use `defaultInferenceEndpoints.ELSER`?